src/RequestHeaderParser.php

```diff
@@ -103,8 +103,12 @@ private function parseRequest($headers)
 
         if ($this->remoteSocketUri !== null) {
             $remoteAddress = parse_url($this->remoteSocketUri);
-            $serverParams['REMOTE_ADDR'] = $remoteAddress['host'];
-            $serverParams['REMOTE_PORT'] = $remoteAddress['port'];
+
+            // port won't be set for unix domain sockets
+            if (isset($remoteAddress['port'])) {
+                $serverParams['REMOTE_ADDR'] = $remoteAddress['host'];
+                $serverParams['REMOTE_PORT'] = $remoteAddress['port'];
+            }
```

Comment:
But the `REMOTE_ADDR` should still be set, no? I guess the port could just be set to `0`.

Reply:
In case of OSX I get `________` as `REMOTE_ADDR` so I've removed both.